cogs/reaction.py

```diff
@@ -50,69 +65,44 @@ async def on_raw_reaction_add(self,payload):
         if payload.emoji.name=='🇩':
             async with self._lock:
                 new_roles = {r.id for r in member.roles[1:]}
-                new_roles -= set(subject_roles)
-                new_roles.discard(780875583214321684)  # Foundational
-                new_roles.add(780879103237947453)  # Foundational Alumni
-                new_roles.add(diploma)
+                new_roles -= set(self.foundational_subject_roles)
```

Comment:
```suggestion
                new_roles -= self.foundational_subject_roles
```

And a few other redundant casts that actually create a new object

Reply:
Thanks a lot for the review 🙇🏻‍♂️ , I don't know how I missed them 😂 , fixed them in the next commit.